### PR TITLE
SRVCOM-2431 Add versions, history, bug reporting for Serverless to templates

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
Main branch only

Issue:
[SRVCOM-2431](https://issues.redhat.com//browse/SRVCOM-2431)

Link to docs preview:

Additional info:

Adds support for standalone serverless docs, including versions, history and bug reporting

### CODE REVIEW

File: _templates/_page_openshift.html.erb [hunk 1]
```diff
@@ -156,31 +167,36 @@
               <option value="3.66">3.66</option>
               <option value="3.65">3.65</option>
             </select>
-        <% else %>
```
Comment (aireilly) [inline _templates/_page_openshift.html.erb:159]: Why is this removed?

**Review by aireilly — APPROVED** [review]
changes lgtm - though I have not previewed the output.

**Comment by gabriel-rh** [comment]
image

image
